src/plugins/external_tool_support/src/blast_plus/FormatDBSupport.cpp

```diff
@@ -113,13 +95,8 @@ void FormatDBSupport::sl_runWithExtFileSpecify(){
     //Check that formatDB or makeblastdb and tempory folder path defined
```

Comment:
Remove formatDB mentioning from the comment. Also fix a typo in word "temporary".